### PR TITLE
New Adapter: Bematterfull

### PR DESCRIPTION
**Pull request by BeMatterFull-Prebid**

### CODE REVIEW

**Comment by BeMatterFull-Prebid** [comment]
Docs PR: https://github.com/prebid/prebid.github.io/pull/4812

File: static/bidder-info/bematterfull.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+endpoint: "http://prebid-srv.mtflll-system.live/?pid={{.SourceId}}&host={{.Host}}"
```
Comment (onkarvhanumante) [inline static/bidder-info/bematterfull.yaml:1]:
```
 ~ % curl -i http://prebid-srv.mtflll-system.live
HTTP/1.1 200 OK
Server: nginx
Date: Mon, 21 Aug 2023 07:06:46 GMT
Transfer-Encoding: chunked
Connection: keep-alive
```

endpoint is reachable

File: static/bidder-info/bematterfull.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+maintainer:
+  email: "adops@bematterfull.com"
```
Comment (onkarvhanumante) [inline static/bidder-info/bematterfull.yaml:3]: Prebid team has sent mail for verification of maintainer email address

File: exchange/adapter_builders.go [hunk 3]
```diff
@@ -234,6 +235,7 @@ func newAdapterBuilders() map[openrtb_ext.BidderName]adapters.Builder {
 		openrtb_ext.BidderAxis:              axis.Builder,
 		openrtb_ext.BidderAxonix:            axonix.Builder,
 		openrtb_ext.BidderBeachfront:        beachfront.Builder,
+		openrtb_ext.BidderBematterfull:      bematterfull.Builder,
 		openrtb_ext.BidderBeintoo:           beintoo.Builder,
```
Comment (onkarvhanumante) [inline exchange/adapter_builders.go:239]: should maintain alpha sorted order here,  `openrtb_ext.BidderBematterfull` should be after 		`openrtb_ext.BidderBeintoo`

```
openrtb_ext.BidderBeintoo:           beintoo.Builder,
openrtb_ext.BidderBematterfull:      bematterfull.Builder,
```

**Comment by onkarvhanumante** [comment]
@BeMatterFull-Prebid  requesting to resolve conflicts on PR

**Review by onkarvhanumante — APPROVED** [review]

**Review by gargcreation1992 — APPROVED** [review]

**Pull request merged**